### DISCUSSION TITLE
tint2 instead of i3bar how to setup?

### DISCUSSION THREAD

**Discussion by gychang** [i3/i3#5857] [Q&A] [answered]
I have installed i3 in a puppylinux distro.  I prefer to have a tint2 bar instead the i3bar, how do I configure this?

**Comment by okraits** [comment, marked as answer]
You simply need to omit the `bar` block in the i3 config and start `tint2`, either from the i3 config by using `exec`/`exec_always` or from `~/.xsession` or `~/.xinitrc`.
